projects/components/src/action-menu/extension-actions/entity-action-extension.component.ts

```diff
@@ -0,0 +1,76 @@
+/*!
+ * Copyright 2020 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+/**
+ * Any entry in an entity action menu.
+ */
+import { Observable } from 'rxjs';
+
+/**
+ * This represents a menu item that can be clicked to perform an action on an entity (e.g. VM).
+ */
+export interface EntityActionExtensionMenuItem {
+    /**
+     * This is a unique URN identifying the action. This is so the component can be notified which action was requested.
```

Comment:
Please remove "This is" from your comments. You should always avoid repetitive text that doesn't add any information